cluster-state-service/canary/pkg/logger/logger.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2016-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why can't you depend on the `logger` package that's already there in `cluster-state-service`? Just asking why do we have duplicated code here?